kits/receipt-budget-tracker/apps/.env.example

```
@@ -0,0 +1,10 @@
+# Lamatic API Configuration
+LAMATIC_API_KEY=your_lamatic_api_key_here
+LAMATIC_PROJECT_ID=your_lamatic_project_id_here
+LAMATIC_API_URL=https://api.lamatic.ai
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAMATIC_API_URL key should go before the LAMATIC_PROJECT_ID key

(UnorderedKey)